airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -552,3 +638,66 @@ def update_yaml_file(filename: str, callback: Callable):
         updated, config = callback(config)
         if updated:
             write_yaml_config(config, filename)
+
+    def clean_tmp_tables(
+        self,
+        destination_type: DestinationType,
+        test_type: str = NORMALIZATION_TYPE_TEST[0],
+        tmp_folders: list = None,
+        git_versioned_tests: list = None,
+    ):
+
+        path_to_sources: str = "/models/generated/sources.yml"
+        test_folders: list = []
+        source_files: list = []
+        schemas_to_remove: list = []
+        macro_name: str = ""
+
+        # choose macro, based on destination_type, for future extendibility
+        # other destination-specific macros could be added by the path:
+        # dbt-project-template/macros/clean_tmp_tables.sql
+        if destination_type == DestinationType.REDSHIFT:
+            macro_name = "redshift__clean_tmp_tables"
```

Comment:
Why is this useful?

Can't you define a clean_tmp_tables macro for default target (maybe that does a no-op?) and then override it for each destination in the macro folder, so you don't need to check anything on python side?

Reply:
Done.